### PR TITLE
Update Prow components to v20191219-ecbeba384

### PR DESCRIPTION
**Pull request by mthssdrbrg**
Relevant commits extracted from #85 to get the current setup into some kind of state that actually works (as things are failing due to `kustomize` version and changes to `checkconfig` (or Prow components not entirely sure)).

### CODE REVIEW

**Comment by mthssdrbrg** [comment]
`/test all`

File: config/config.yaml [hunk 1]
```diff
@@ -144,11 +144,3 @@ presets:
     env:
       - name: GOLANG_JUNIT
         value: "true"
-
-  # Istio set this inside their CI builder image and their scripts rely on this
```
Comment (theobarberbany) [inline config/config.yaml]: Do we no longer need to run Istio CI stuff ?

**Review by theobarberbany — DISMISSED** [review]
Other than the why no istio env var question, all looks good.

File: config/config.yaml (hunk 1 above)
Comment (mthssdrbrg) [reply config/config.yaml]: I believe Prow now sets this automatically (vaguely remember hearing something along those lines), so we don't need to have an explicit preset for it.

Comment (theobarberbany) [reply config/config.yaml]: Ah, ok! I'm happy with this then :)

Comment (mthssdrbrg) [reply config/config.yaml]: Found the commit that added it https://github.com/kubernetes/test-infra/commit/c6f0e08ab3. Initially thought that the value of the variable didn't matter, but it actually seems like it does: https://github.com/pusher/istio/blob/pusher-release-1.2/prow/lib.sh#L34

**Review by theobarberbany — APPROVED** [review]
/lgtm

**Pull request merged**